### PR TITLE
Base structure of website frontend and update page form

### PR DESCRIPTION
**Pull request by kazutan1230**
### What does this PR do?

I updated a basical structure of this web site.

- Landing page
I added links to upload page and download page.

- update page
I made upload form using react-form-hook library,
but haven't developed submit button function yet.

- download page
I've develop nothing, just added page.

### How did you verify your code works?

By localhost server and github workflows.

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -5,29 +5,56 @@ export default function Home() {
...
+              -&gt;
+            </span>
+          </h2>
+          <p className={'m-0 max-w-[30ch] text-sm opacity-50'}>
+            [派遣従業員向け]障がい者手帳画像をアップロードします。
+          </p>
+        </a>
+
+        <a
+          href="/upload"
```
Comment (Marukome0743) [inline app/page.tsx:41]: I wonder `href="/download"` is correct, right?

File: app/page.tsx [hunk 2]
```diff
@@ -5,29 +5,56 @@ export default function Home() {
...
           </a>
         </div>
       </div>
 
+      <div className="mb-32 grid text-center lg:max-w-5xl lg:w-full lg:mb-0 lg:grid-cols-2 lg:text-left">
+        <a
+          href="/upload"
+          className="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
+          // target="_blank"
+          // rel="noopener noreferrer"
```
Comment (Marukome0743) [inline app/page.tsx:27]: Remove comment out codes.

File: app/upload/layout.tsx
Comment (Marukome0743) [inline app/upload/layout.tsx:1]: Remove this file, also.

File: app/upload/style.css
Comment (Marukome0743) [inline app/upload/style.css:1]: Remove

**Review by Marukome0743 — CHANGES_REQUESTED** [review]
There are some unnecessary files.
I'll make another branch if you want to commit the all of them.
Then push there.

File: compose.yml
[the remark this answers is not in the document]
Comment (kazutan1230) [reply compose.yml:1]: Thank you.
I'll delete.

File: app/download/layout.tsx
[the remark this answers is not in the document]
Comment (kazutan1230) [reply app/download/layout.tsx:1]: Thank you.
I'll delete.

File: app/upload/layout.tsx
Comment (kazutan1230) [reply app/upload/layout.tsx:1]: Thank you.
I'll delete.

File: app/page.tsx (hunk 1 above)
Comment (kazutan1230) [reply app/page.tsx:41]: You're right, I'll fix.

File: app/page.tsx (hunk 2 above)
Comment (kazutan1230) [reply app/page.tsx:27]: okay.

File: app/download/style.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+.required::after {
+  content: "必須";
+  color: #ffffff;
+  background: #cc0000;
+  font-size: 0.8em;
+  padding: 0.3em;
+  border-radius: 0.5em;
+  margin-left:0.3em;
+}
```
[the remark this answers is not in the document]
Comment (kazutan1230) [reply app/download/style.css:37]: okay.

File: app/upload/style.css
Comment (kazutan1230) [reply app/upload/style.css:1]: okay.

File: components/uploadForm.tsx
[the remark this answers is not in the document]
Comment (kazutan1230) [reply components/uploadForm.tsx:1]: I'll delete it.

**Comment by kazutan1230** [comment]
@Marukome0743
I pushed the version for your comment.

**Comment by HaradaTakayuki** [comment]
good

**Review by HaradaTakayuki — APPROVED** [review]
good

**Review by Marukome0743 — APPROVED** [review]
AnimeKuboSanGIF

**Pull request merged**